src/map/map.ts

```diff
@@ -678,13 +678,13 @@ export class PropertiesMap {
         // make a copy of the default layout
         const layout = JSON.parse(JSON.stringify(DEFAULT_LAYOUT)) as typeof DEFAULT_LAYOUT;
         // and set values speific to the displayed dataset
-        layout.xaxis.title = this._options.x.property.value;
-        layout.yaxis.title = this._options.y.property.value;
+        layout.xaxis.title = this._title(this._options.x.property.value);
+        layout.yaxis.title = this._title(this._options.y.property.value);
         layout.xaxis.type = this._options.x.scale.value;
         layout.yaxis.type = this._options.y.scale.value;
-        layout.scene.xaxis.title = this._options.x.property.value;
-        layout.scene.yaxis.title = this._options.y.property.value;
-        layout.scene.zaxis.title = this._options.z.property.value;
+        layout.scene.xaxis.title = this._title(this._options.x.property.value);
+        layout.scene.yaxis.title = this._title(this._options.y.property.value);
+        layout.scene.zaxis.title = this._options.z.property.value; // using this._title(this._options.z.property.value) here seems to cause troubles
```

Comment:
I guess the issue is that `this._options.z.property.value` can be set to the empty string `''` when in 2D mode, and that `this._title` does not account for it.

Reply:
```suggestion
        layout.scene.zaxis.title = this._title(this._options.z.property.value);
```

given that this._title should handle this, can we implement as with x/y axis?

Reply:
This is still TBD